test_schemas/en/test_new_routing_and.json

```diff
@@ -0,0 +1,173 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "language": "en",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.3",
+    "survey_id": "001",
+    "title": "Test Routing And",
+    "theme": "default",
+    "description": "A test survey for routing based on the and operator",
```

Comment:
I would emphasis the operators i.e `and`. The same applies to the other schemas.